README.md

```diff
@@ -22,26 +41,192 @@ allprojects {
+                    }
+                }
+            }
+```
+
+## Track a custom event
+
+Once you've identified a customer, you can use the `track` method to capture customer activity and send it Customer.io, optionally supplying event data with your request
+
+```kotlin
+CustomerIO.instance().track(
+                name = "logged_in",
+                attributes = mapOf("ip" to "127.0.0.1")
+            ).enqueue { result ->
+                when (result) {
+                    is ErrorResult -> {
+                        // Handle error
+                    }
+                    is Success -> {
+                        // Handle success
+                    }
+                }
+            }
+```
+
+`attributes` accepts the following data types:
+- Primitives (int, float, char...) and their boxed counterparts (Integer, Float, Character...).
+- Arrays, Collections, Lists, Sets, and Maps
+- Strings
+- Enums
+
+## Stop identifying a customer
+In your app you may need to stop identifying a profile in the Customer.io SDK. There are 2 ways to do that:
+
+1. Call the `identify()` function which will stop identifying the previously identified profile (if there was one) and remember this new profile.
+2. Use `clearIdentify()` to stop identifying the previously identified profile (if there was one). 
+
+```kotlin
+CustomerIO.instance().clearIdentify()
+```
+
+# Push notifications 
+Want to send push notification messages to your customer's devices? Great!
+
+The Customer.io SDK supports sending push notifications via FCM for now. 
+
+## Getting Started FCM 
+
+### Dependency
+Push messaging artifact is needed in order to use push notification features
+```groovy
+implementation 'io.customer.android:messaging-push-fcm:<version-here>'
+```
+
+The artifact adds a `FirebaseMessagingService` to the app manifest automatically, so you don't have to do any extra setup to handle incoming push messages.
+
+In case, your application implements its own `FirebaseMessagingService`, make sure on  `onMessageReceived` and `onNewToken` method calls you additionally call CustomerIOFirebaseMessagingService.onMessageReceived and CustomerIOFirebaseMessagingService.onNewToken, respectively:
+
+```kotlin
+class FirebaseMessagingService : FirebaseMessagingService() {
+
+ override fun onMessageReceived(message: RemoteMessage) {
+        val handled = CustomerIOFirebaseMessagingService.onMessageReceived(this, message)
+        if (handled) {
+            logger.breadcrumb(this, "Push notification has been handled", null)
+        }
+ }
+ 
+ override fun onNewToken(token: String) {
+     CustomerIOFirebaseMessagingService.onNewToken(token) { result ->
+                 when (result) {
+                     is ErrorResult -> {
+                         // Handle error
+                     }
+                     is Success -> {
+                         // Handle success
+                     }
+                 }
+             }
+     }
+}
+```
+
```

Comment:
Somewhere in the push notifications section of the docs we should mention our default Android Notification channel and our plans to allow you to customize it in the future.